mas/src/mas.js

```diff
@@ -10,14 +8,40 @@ const features = searchParams.get('features');
 const envName = isStage ? 'stage' : 'prod';
 const commerceEnv = isStage ? 'STAGE' : 'PROD';
 
+function loadScript(url) {
```

Comment:
this should not be needed. 
We should import the file not load it this way.

Reply:
removed